final_prep/README.md

```diff
@@ -45,30 +45,39 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+
+[Week 1] <img width="1257" alt="Screen Shot 2022-02-13 at 3 58 28 PM" src="https://user-images.githubusercontent.com/48334178/153779098-3207a456-c405-4d41-9070-2bb352a709cd.png">
+
 - `Add Week 2 Screenshot Here`
+
+[Week 2]<img width="1269" alt="Screen Shot 2022-02-13 at 3 58 39 PM" src="https://user-images.githubusercontent.com/48334178/153779137-ce6fe9bd-eaf7-491f-bd2f-55049099105e.png">
+
 - `Add Week 3 Screenshot Here`
 
+[Week 3]<img width="1267" alt="Screen Shot 2022-02-13 at 3 58 49 PM" src="https://user-images.githubusercontent.com/48334178/153779161-748d221b-a38d-4ce7-b10e-6672258a5f08.png">
+
+
 ## Mentorship Prep
 
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
 
 - [ ] [Create a profile on Turing Community Connect by following these directions](https://docs.google.com/document/d/1vpyKGu92l1HGkJzULNcyyE72946f4QO1DhQgIz3v1E0/edit?usp=sharing)
 - [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+  - [ ] Add link to your gist here: [Mentorship Gist](https://gist.github.com/philmarcu/1d8cda649b75af5c8fd92e0742261034)
 
 ## Lesson Prep
 
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
 
 - [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+  - [ ] Add a link to your gist here: [Lesson Prep](https://gist.github.com/philmarcu/195fd0b4aac417974b2e41df82f87032)
 
 ## Group Work Prep
 
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
 
 - [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
```

Comment:
I liked your idea of breaking down a timeline, so that everyone is on the same page about where you should be when as you're working through a big project.